browser/importer/brave_external_process_importer_host.h

```diff
@@ -1,32 +1,33 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
```

Comment:
2020? :)

Reply:
I did this intentionally because this file was created last year.

Reply:
Hmm, should we use 2020 if we change license sentence?

Reply:
I think it's OK as-is 😄